modules/checklist/steps/step-base.php

```diff
@@ -11,6 +11,13 @@
 }
 
 abstract class Step_Base {
+	const TIER_CORE = 'core';
+	const TIER_ESSENTIAL = 'essential';
+	const TIER_ADVANCED = 'advanced';
+	const TIER_EXPERT = 'expert';
+	const TIER_AGENCY = 'agency';
```

Comment:
Do we maybe have these already another place in the code base?

Reply:
`modules/connect/module`
Is this global enough ti use?

Reply:
I would maybe extract to some common area and re-use it.

Reply:
I created a `constants` file in the `core/utils` folder